src/widget/weffectparameterknob.h

```diff
@@ -4,6 +4,11 @@
 #include "widget/wknob.h"
 #include "effects/effectparameterslot.h"
 
+// This is used for effect parameter knobs with dynamic
```

Comment:
Thanks for adding these comments to the code. We should have brief descriptions like these for most if not all classes in the code. It would make it a lot easier for new people to get started contributing code.